mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -42,6 +42,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 createUpdatePacket ()Lnet/minecraft/class_2596;
```

Comment:
If you don't mind, I'd prefer the one from #2692

Reply:
Yep, that one is definitely better.